pkg/kubectl/cmd/apply.go

```diff
@@ -78,7 +78,14 @@ var (
 		kubectl apply -f ./pod.json
 
 		# Apply the JSON passed into stdin to a pod.
-		cat pod.json | kubectl apply -f -`)
+		cat pod.json | kubectl apply -f -
+
+		# Note: --prune is still in Alpha
+		# Apply the configuration in manifest.yaml and delete all the other resources that are not in the file and match label app=nginx.
+		kubectl apply --prune -f manifest.yaml -l app=nginx
```

Comment:
Beside the prune part, this command will only apply objects in manifest.yaml that match label `app=nginx`. Feel like this comment does not say it clearly.